encoder/vaapiencoder_h264.cpp

```diff
@@ -1353,11 +1362,39 @@ bool VaapiEncoderH264::ensureSlices(const PicturePtr& picture)
 Encode_Status VaapiEncoderH264::encodePicture(const PicturePtr& picture)
 {
     Encode_Status ret = ENCODE_FAIL;
+
     SurfacePtr reconstruct = createSurface();
     if (!reconstruct)
         return ret;
     {
         AutoLock locker(m_paramLock);
+
+#ifdef __BUILD_GET_MV__
```

Comment:
can we hiddle all this details in bool editMvBuffer(void*& buffer, uint32_t size);?